Makefile

```diff
@@ -25,43 +25,45 @@ ifeq (,$(filter %$(BUILD_META),$(TAG)))
 $(error TAG ${TAG} needs to end with build metadata: $(BUILD_META))
 endif
 
+REPO ?= rancher
+IMAGE = $(REPO)/hardened-flannel:$(TAG)
+BUILD_OPTS = \
+	--platform=$(TARGET_PLATFORMS) \
+	--build-arg PKG=$(PKG) \
+	--build-arg SRC=$(SRC) \
+	--build-arg TAG=$(TAG:$(BUILD_META)=) \
+	--build-arg K3S_ROOT_VERSION=$(K3S_ROOT_VERSION) \
+	--tag "$(IMAGE)"
+
 .PHONY: image-build
 image-build:
 	docker build \
+		$(BUILD_OPTS) \
 		--pull \
```

Comment:
I saw that in some repos we are using --pull and --load whereas in this one we only have --pull, could that be a problem?

Reply:
This should not cause any problems. The image-build make target is only used in local development, and I left the --pull option in place if it was already there.